test/e2e/flask/solana/check-balance.spec.ts

```diff
@@ -6,7 +6,11 @@ describe('Check balance', function (this: Suite) {
   this.timeout(300000);
   it('Just created Solana account shows 0 SOL when native token is enabled', async function () {
     await withSolanaAccountSnap(
-      { title: this.test?.fullTitle(), showNativeTokenAsMainBalance: true },
+      {
+        title: this.test?.fullTitle(),
+        showNativeTokenAsMainBalance: true,
+        mockCalls: true,
```

Comment:
I expect we'll need `mockCalls` enabled elsewhere as well. I'm not sure why they'd ever be disabled 🤔 

Maybe if a test was using its own? But that doesn't seem to be true of the ones I'm finding with this disabled. And even if that was the case, I'd expect `true` to be default so we don't make it too easy to accidentally leave things unmocked

Reply:
Oh I see that `withSolanaAccountsnap` does some mocking even when `mockCalls` is `false`.

Perhaps we could instead include this mock irrespective of whether `mockCalls` is set

Reply:
yeah I was also a bit unsure why the test was not using mocks  :thinking: and unsure in which case we would want this disabled.
Maybe in a case where we want to pass different mocks? though this current approach doesn't seem ideal for that :thinking: 

mmm for the current fix, would it be fine if I set the mockCalls as true by default?

Reply:
just saw your latest comment, I've pushed some new changes

Reply:
mockCalls to false is used for negative cases (basically, validation error flows). After reading the slack thread, I could change the approach and even in that case, mock it in a different way

Reply:
This specific mock may be OK for both cases, since both cases were already the same before now (both using the same unmocked token metadata from the live API).